### ISSUE TITLE
Introduce `onReady` instead of `onChange`

### ISSUE THREAD

**Issue by pomek** [ckeditor/ckeditor5-dev-compiler#64]
It's quite complicated to use `onChange` property as a callback which is called when compiler ends working.

We use `onChange` property in `ckeditor5-dev-tests` to run tests when [the compiler will finish the whole compilation](https://github.com/ckeditor/ckeditor5-dev-tests/blob/master/lib/tasks.js#L79). As I mentioned, it's complicated because we call `onChange` callback very often. Too often. Would be better to run this callback once - when the compiler finishes his work.

Most probably if we will introduce the `onReady` callback, we will be able to remove `onChange`.

**Comment by Reinmar** [comment]
Yeah, I thought that `onChange` will be enough, but it's hard to handle it so it's better to expose a more useful callback.
